main.cpp

```diff
@@ -300,6 +382,26 @@ float Bank::wireMoney(float amountOfMoneyWired)
     return amountOfMoneyWired;
 }
 
+float Bank::convertToCanadianDollar(float dollarValue)
+{
+    float usdValue = 1.f;
+    float exchangeRate = 0.74f;
+    float convertedValue = usdValue / exchangeRate;
+    float totalCash = 1452584.63f;
+    float markupRate = convertedValue * 100.f;
+    float conversionValue = totalCash * convertedValue + markupRate;
+    bool totalConverted = true;
+
+    while(dollarValue <= 1)
+    {
+        ++conversionValue;
+        std::cout << "Calculated conversion value: " << conversionValue << std::endl;
+        if(totalConverted)
+        break;
```

Comment:
Indent the body of any `if` blocks, even if they're a single statement/single line
```
if( condition )
    doSomething();
```
not:
```
if( condition )
doSomething();
```
- [ ] revise